artifactory_test.go

```diff
@@ -297,7 +298,55 @@ func TestAqlFindingItemOnRoot(t *testing.T) {
 	isExistInArtifactory(tests.GetSingleFileCopy(), searchPath, t)
 	cleanArtifactoryTest()
 }
+func TestExitCodeFlow(t *testing.T) {
```

Comment:
I suggest renaming this function to **TestExitCode**.

Reply:
Fixed.